### DISCUSSION TITLE
Duda sobre Buyer Persona

### DISCUSSION THREAD

**Discussion by ErickBarreraO** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#120] [Q&A] [answered]
Hola compañeros buenas noches, haciendo la practica 2 vino a mi mente una idea.
Cuando un cliente hace una solicitud para desarrollar algún proyecto, hasta donde entiendo, parte del desarrollo es la definición del buyer persona y está va enfocada específicamente para los fines de ese proyecto.
Mi duda va en el sentido de ¿Se puede reciclar un buyer persona que cuadre con otro otro proyecto para otro cliente? No se si esto sea ético, funcional, etc o estoy malentendiendo el tema.  
Gracias compañeros

**Comment by doguedogue** [comment, marked as answer]
Se podría dar el caso si son proyectos parecidos, pero como sabes cada proyecto es único y su _buyer_ persona siempre es muy especifico y particular, por la época o algo distinto, lo que tu quieras, aunque si tenga similitudes.
